doc/gvmd.8

```diff
@@ -182,9 +185,6 @@ Time out tasks that are more than TIME minutes overdue. -1 to disable, 0 for min
 \fB--secinfo-commit-size=\fINUMBER\fB\f1
 During CERT and SCAP sync, commit updates to the database every NUMBER items, 0 for unlimited.
 .TP
-\fB--slave-commit-size=\fINUMBER\fB\f1
-During slave updates, commit after every NUMBER updated results and hosts, 0 for unlimited.
-.TP
```

Comment:
Has this been removed by accident or on purpose? Maybe it belongs to another PR?

Reply:
This option is supposed to be removed. I think it was added back in by accident in a backport or it was just forgotten before.